.github/workflows/mysql.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           override: true
 
       - name: Set up protoc
-        uses: arduino/setup-protoc@v2.0.0
+        uses: arduino/setup-protoc@v2.1.0
```

Comment:
Perhaps we can just make it @v2 so we don't get nagged about minor version updates?